agecheck-js.js

```diff
@@ -9,11 +9,29 @@
   //Constant for Expiry date in days
   var ageCheckDisplayedExpiryDays = 28;
 
+  // Default options
+  var defaultOptions = {
+    colours: {
+
+    }
+  };
+
   //Element references
   var acModal = document.getElementById('acModal');
   var acYesBtn = document.getElementById('acYesBtn');
   var acNoBtn = document.getElementById('acNoBtn');
 
+  function mergeObjects() {
+    var args = Array.prototype.slice.call(arguments);
+    var mergedObj = {};
+    for(var arg in args) {
+      for(var propName in arg) {
+        mergedObj[propName] = arg[propName];
```

Comment:
Might want to consider expanding this function, as currently it doesn't merge the nested props, it just does them at this level. It'll do for now, but could cause issues down the line I guess.